pyodide-build/pyodide_build/tests/test_pywasmcross.py

```diff
@@ -2,13 +2,13 @@
 
 import pytest
 
-from pyodide_build.pywasmcross import handle_command_generate_args  # noqa: E402
-from pyodide_build.pywasmcross import replay_f2c  # noqa: E402
 from pyodide_build.pywasmcross import (
     BuildArgs,
     calculate_exports,
     get_cmake_compiler_flags,
     get_library_output,
+    handle_command_generate_args,  # noqa: E402
+    replay_f2c,  # noqa: E402
```

Comment:
But probably these ones should be fixed by adding an `__all__` to pywasmcross.py?

Reply:
I think that ignoring E402 is not needed anymore. It is about "module level import not at top of file" but now the import is done in top of file.